docs/userguides/highriskemployee.md

```diff
@@ -32,19 +32,16 @@ To remove a user from the High Risk Employee list:
 sdk.detectionlists.high_risk_employee.remove(user_id)
 ```
 
-For complete details, see
- [High Risk Employee](../methoddocs/detectionlists.html#high-risk-employee).
-
-
-##
- Add or Remove Risk Factors From Users
+## Add or Remove Risk Factors From Users
```

Comment:
This case is applicable for both `high risk employees` and `departing employees`.  Right now its found in this `High Risk Employee` section.  

Would it make more sense to just create a user guide section titled `Detection Lists` that contains this, and the other `high risk employee`/`departing employee` guides instead of having them separate as they are now?

Reply:
On cool, I didn't even know that!  It was in this section but otherwise I couldn't tell that it was only for `high_risk_employees`. Maybe I'll just add a note that specifies that in this section.